projects/graph/src/graph.py

```diff
@@ -1,9 +1,53 @@
 """
 Simple graph implementation compatible with BokehGraph class.
 """
-
+import random
+import math
 
 class Graph:
     """Represent a graph as a dictionary of vertices mapping labels to edges."""
     def __init__(self):
-        pass  # TODO
+        self.vertices = {}
+    # def add_vertex(self, v):
+    #     self.vertices[v] = set()
+    def add_vertex(self, v):
+        self.vertices[v] = Vertex(v)
+    def add_edge(self, v1, v2):
+        self.vertices[v1].edges.add(v2)
+        self.vertices[v2].edges.add(v1)
+    def dfs(self, value, visited=None):
```

Comment:
you're on the right track.  a few pointers:
- we don't need to search if a vertex exists, we want to return every vertices that has a connection to the `value` being passed in
- you don't need to loop thru all the vertices every time you call `dfs` you can start with `self.vertices[value].edges`